server/controllers/questionsControllers.js

```diff
@@ -7,25 +7,24 @@ const Question = {
     const data = req.body;
```

Comment:
consider renaming the variable data. Variable names should be descriptive and meaningful. It should also be indicative of what is being stored